src/screens/Login/ConfirmMnemonic/ConfirmMnemonicContainer.js

```diff
@@ -9,12 +9,15 @@ import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
 import PropTypes from 'prop-types'
 import { createAccount } from '@chronobank/ethereum/redux/thunks'
+import { getEthAccounts } from '@chronobank/ethereum/redux/selectors'
 import { MNEMONIC_LENGTH } from '../../../common/constants/globals'
 import i18n from '../../../locales/translation'
 import ConfirmMnemonic from './ConfirmMnemonic'
 
-const mapStateToProps = () => {
-  return {}
+const mapStateToProps = (state) => {
+  return {
+    accounts: getEthAccounts(state),
```

Comment:
I suggest to rename it to getEthAccountList
Any selector which returns array must have '*List' postfix.